clients/src/main/java/org/apache/kafka/clients/admin/internals/CoordinatorStrategy.java

```diff
@@ -136,8 +136,8 @@ private CoordinatorKey requireSingletonAndType(Set<CoordinatorKey> keys) {
     }
 
     private void ensureSameType(Set<CoordinatorKey> keys) {
-        if (keys.size() < 1) {
-            throw new IllegalArgumentException("Unexpected size of key set: expected >= 1, but got " + keys.size());
+        if (keys.isEmpty()) {
+            throw new IllegalArgumentException("Unexpected size of key set: expected >= 1, but got 0");
```

Comment:
maybe we can say "the keys can't be empty"

Reply:
It's the same really. I just simplified the error message but be a fixed string.